src/main/java/graphql/schema/idl/SchemaGeneratorHelper.java

```diff
@@ -1087,9 +1088,8 @@ private Optional<OperationTypeDefinition> getOperationNamed(String name, Map<Str
         return Optional.ofNullable(operationTypeDefs.get(name));
     }
 
-    private DataFetcher<?> dataFetcherOfLastResort(FieldWiringEnvironment environment) {
-        String fieldName = environment.getFieldDefinition().getName();
-        return new PropertyDataFetcher(fieldName);
+    private DataFetcher<?> dataFetcherOfLastResort() {
+        return SingletonPropertyDataFetcher.singleton();
```

Comment:
This code is not actually used by default - unless GraphqlCodeRegistry had a null default DDF